Makefile.js

```diff
@@ -708,6 +736,51 @@ target.checkLicenses = function() {
     });
 };
 
+target.checkGitCommit = function() {
+    var commitMsgs,
+        failed;
+
+    if (hasBranch("master")) {
```

Comment:
So, can I remove the check for single commit?  Maybe that can be thought through and submitted in a subsequent PR, since it seems more complicated than I first thought.